app/models/manageiq/providers/redfish/physical_infra_manager/physical_server.rb

```diff
@@ -1,7 +1,13 @@
 module ManageIQ::Providers::Redfish
   class PhysicalInfraManager::PhysicalServer < ::PhysicalServer
+    include_concern "Operations"
+
     def self.display_name(number = 1)
       n_('Physical Server (Redfish)', 'Physical Servers (Redfish)', number)
     end
+
+    def kind
```

Comment:
Same concern as https://github.com/ManageIQ/manageiq/pull/18573/files#r270238530.  The method name is too generic.